.github/workflows/canary-build.yml

```diff
@@ -0,0 +1,63 @@
+name: Canary build
+
+# Controls when the action will run. Triggers the workflow on push
+# events but only for the canary branch
+on:
+  push:
+    branches:
+      - canary
+
+# Publishes Canary builds with the dist tag of "canary"
+jobs:
+  # This workflow contains a single job called "publish"
+  publish:
+    name: Canary build publish
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      # Caches NPM files
+      - name: Cache NPM files
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      # Sets up Node v12
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+          registry-url: 'https://npm.pkg.github.com'
+
+      # NPM ci init
+      - run: npm ci
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_PERSONAL_AUTH_TOKEN }}
+
+      # Git config
+      - run: git config --global user.email "beltran.caliz@washpost.com" && git config --global user.name "beltrancaliz"
```

Comment:
can we set this as a different username like a bot? or does it need to be you because using your npm auth token?

Reply:
without setting this it would throw a git error and when i looked it up it needs to be an account that has access to the repo and a personal token

Reply:
ok. this is probably good for now. but might want to make an issue or ticket for setting up bots ... out of scope though